eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionReportWriterTest.java

```diff
@@ -174,4 +183,43 @@ private void assertHeaderIsNotWrittenTwice(File output) throws IOException {
             variantLine = fileInputStream.readLine();
         } while (variantLine != null);
     }
+
+    @Test
+    public void shouldSortReport() throws IOException {
+        // given
+        SubmittedVariant firstVariant = new SubmittedVariant("assembly", TAXONOMY, "project", CONTIG_1, START_1,
+                                                             "reference", "alternate", false);
+        SubmittedVariant secondVariant = new SubmittedVariant("assembly", TAXONOMY, "project", CONTIG_2, START_2,
+                                                              "reference", "alternate", false);
+        SubmittedVariant thirdVariant = new SubmittedVariant("assembly", TAXONOMY, "project", CONTIG_1, START_2,
+                                                             "reference", "alternate", false);
+        SubmittedVariant fourthVariant = new SubmittedVariant("assembly", TAXONOMY, "project", CONTIG_2, START_1,
+                                                              "reference", "alternate", false);
+        List<SubmittedVariant> variants = Arrays.asList(firstVariant, secondVariant, thirdVariant, fourthVariant);
+
+        // when
+        accessionReportWriter.write(mockWrap(variants));
+
+        // then
+        BufferedReader fileInputStream = new BufferedReader(new InputStreamReader(new FileInputStream(output)));
+        String line;
+        while ((line = fileInputStream.readLine()) != null) {
+            if (!line.startsWith("#")) {
+                break;
+            }
+        }
```

Comment:
You could do just:

```java
while ((line = fileInputStream.readLine()) != null && line.startsWith("#"));
```

This is fine because Java [guarantees sub-expressions to be evaluated from left to right](https://docs.oracle.com/javase/specs/jls/se7/html/jls-15.html#jls-15.7).